doc/modules/clustering.rst

```diff
@@ -498,6 +498,17 @@ geometrical shape.
 |face_kmeans|                          |face_discretize|
 =====================================  =====================================
 
+Spectral Clustering Graphs
+--------------------------
+
+Spectral Clustering can also be used to cluster graphs by their spectral embeddings.
+In this case, the affinity matrix is the adjacency matrix of the graph, and SpectralClustering
+is initialized with `affinity='precomputed'`:
+
+```
```

Comment:
This looks like markdown, rather than restructured text. Check how we do this elsewhere.